src/templates/Section.tsx

```diff
@@ -1,17 +1,111 @@
 import React from 'react'
+import { IArticle, IMostReadArticle } from '../types'
```

Comment:
add a space between `import React from 'react'` and `import { IArticle, IMostReadArticle } from '../types'` to make the distinction between importing from external packages and importing from local files